### PR TITLE
feat: add boto to docker image for aws usage

### PR DESCRIPTION
**Pull request by sabotagebeats**
### What I did

added boto to docker image for better integrations with AWS

### How I did it

add pip install boto to the dockerfile

### How to verify it

run the new dockerfile and then import boto3 in a script or in the python console

### Checklist

- [ ] All changes are completed
- [ ] New test cases have been added
- [ ] Documentation has been updated

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -30,6 +30,8 @@ RUN pip install --upgrade pip \
 # Fix RLP installation issue
     && pip uninstall rlp --yes \
     && pip install rlp==3.0.0 \
+# Add boto for aws compatibility
```
Comment (unparalleled-js) [inline Dockerfile:33]: I have some thought here... Discussing on Telegram / Discord.

**Comment by sabotagebeats** [comment]
After discussion with @unparalleled-js It is possible we should do this in another way possibly creating an alternative docker image that has this tooling since Someone running local docker containers on their computer may not care about AWS and it may bloat the image unnecessarily.

**Comment by helloibis** [comment]
> After discussion with @unparalleled-js It is possible we should do this in another way possibly creating an alternative docker image that has this tooling since Someone running local docker containers on their computer may not care about AWS and it may bloat the image unnecessarily.

Docker has `--build-arg` where you can pass inputs in when building a Docker image, would that be an option maybe that they could toggle whether they want boto?

**Comment by unparalleled-js** [comment]
> Docker has `--build-arg` where you can pass inputs in when building a Docker image, would that be an option maybe that they could toggle whether they want boto?

There is also an ARG on the Dockerfile that you can use `--build-arg` for and point to a custom requirements file.

**Comment by helloibis** [comment]
> There is also an ARG on the Dockerfile that you can use `--build-arg` for and point to a custom requirements file.

That'd be really cool!

**Comment by sabotagebeats** [comment]
Closed because it is much better for image size to put this in a custom docker file using apeworx/ape as a base.

**Pull request closed without merging**